lib/templ.camlp5.ml

```diff
@@ -1377,7 +1377,7 @@ let rec interp_ast conf ifun env =
         let sl = List.map (eval_ast env ep) al in String.concat "" sl
     | None ->
         match f, vl with
-          "capitalize", [VVstring s] -> Utf8.capitalize s
+        | "capitalize", [VVstring s] -> Utf8.capitalize (Mutil.trim_leading s)
```

Comment:
A capitalize function should only capitalize, but not do anything else!

Reply:
Look at my first comment (Aug 20, 2020) for an explanation of the reason behind this PR.
I agree that systematically trimming leading spaces before capitalise could be harmful.
I suggest to add a second function %trim_capitalize; performing the trim before calling Utf8.capitalize. 
This new function would be used only in the places where it appears necessary, leaving all the other instances of capitalise; alone.


Altogether, I suggest to close this PR, and to look at the problem again when things have settled